demos.md

```diff
@@ -29,3 +29,14 @@ Date: 5 Jul 2018
 Create an OVS bridge and connect a system interface to it.
 
 [![asciicast](https://asciinema.org/a/bv2g1RHADTjokXH4uYFljpNC6.png)](https://asciinema.org/a/bv2g1RHADTjokXH4uYFljpNC6)
+
+## Demo-4: Revert when state change fails
+Date: 25 September 2018
+
+On virtual network interfaces, changing the device speed does not work. This is
```

Comment:
Could you please start by stating the main operation without entering the details too much? For someone who is not in to the details, it will be hard to understand.
Something like: "Attempting to change the speed of a virtual interface silently fails (execution passes without error, but the current state is unchanged) and when used with the verification check, the end result is a full revert to the original state".
Hope you can rephrase that better.

BTW, if you can change the date to the date we actually gave the demo (3 weeks ago) it will be nice.

Reply:
hope this is now better